### PR TITLE
Close #450 add homepage background models and add background to admin 

### LINKED ISSUE #450: Add homepage background models and add background to admin

### PR DESCRIPTION
**Pull request by leak17**
Screenshot from 2023-07-06 15-53-51
Screenshot from 2023-07-06 15-54-18
Screenshot from 2023-07-06 16-21-05
Screenshot from 2023-07-06 15-54-52

### CODE REVIEW

**Comment by channainfo** [comment]
@leak17 add demo screens

**Comment by leak17** [comment]
> @leak17 add demo screens

It's just a scope bong; It doesn't have any UI on it.

File: app/views/spree/admin/homepage_background/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="form-group">
+  <%= f.field_container :title do %>
+    <%= f.label :title %>
+    <%= f.text_field :title, class: 'form-control' %>
+  <% end %>
+  <%= f.field_container :redirect_url do %>
```
Comment (theachoem) [inline app/views/spree/admin/homepage_background/_form.html.erb:6]: Remove redirect url.

File: lib/spree_cm_commissioner/test_helper/factories/homepage_background_factory.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FactoryBot.define do
+  factory :cm_homepage_background, class: SpreeCmCommissioner::HomepageBackground do
+    title         {FFaker::Name.unique.name}
+    redirect_url  {FFaker::Internet.http_url}
```
Comment (theachoem) [inline lib/spree_cm_commissioner/test_helper/factories/homepage_background_factory.rb:4]: Redirect url

**Pull request merged**